### DISCUSSION TITLE
Force skip despite change of a function

### DISCUSSION THREAD

**Discussion by tagteam** [ropensci/targets#870] [General]
is it possible to force skip some targets? context: I have run a lot of different simulation scenarios and then found a bug in one of the functions. However, the bug only affects very few of the scenarios. Since the bug fix changed the function, targets recognizes all scenarios are outdated as expected. I would like to force skip the targets that (I know) are not affected. Is this possible? I am rather new to targets, sorry if I have overlooked the obvious solution.

**Comment by wlandau** [comment]
I’d have a look at the tar_cue() function, e.g. tar_target(…, cue = tar_cue(depend = FALSE)

**Reply by tagteam** [reply]
yes, this helps. thanks a lot!
